js/send-mail.js

```diff
@@ -122,7 +122,8 @@ $(function() {
 			},
 			type: 'POST',
 			success:function() {
-				Mail.UI.messageView.setMessageFlag(messageId, 'answered', true);
+				// Reload the message list to display the answered flag.
+				Mail.UI.messageView.loadMessages();
```

Comment:
not necessary to call the server again - we know the message was flagged: so please simply update the flag - as done before